components/public-api/gitpod/experimental/v1/cell_stats.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service CellStatsService {
```

Comment:
@easyCZ @csweichel What do you think about such API?

Reply:
We can call it TelemetryService instead, or UserStatsService, or just StatsService

Reply:
I can live with CellStatsService. I'd prefer `StatsService` but i'm fine with either